data-import/guide-import-csv/guide-import-csv.adoc

```diff
@@ -10,282 +10,391 @@
-* Blog Series on LOAD CSV http://jexp.de/blog/2014/06/load-csv-into-neo4j-quickly-and-successfully/[Part 1], http://jexp.de/blog/2014/10/load-cvs-with-success/[Part 2]
-* http://www.markhneedham.com/blog/?s=load+csv+neo4j[LOAD CSV Tips] by Mark Needham
-// * Example Northwind Dataset on https://github.com/neo4j-contrib/northwind-neo4j[GitHub]
-* Blog Post: http://jexp.de/blog/2014/06/using-load-csv-to-import-git-history-into-neo4j/[Import git-log data into Neo4j]
---
+===== +++<u>LOAD CSV Resources</u>+++
+* link:/developer/desktop-csv-import/[HowTo: Import CSV in Neo4j Desktop]
+* link:/docs/cypher-manual/current/clauses/load-csv/[Cypher Manual: LOAD CSV^]
+* link:/developer/guide-importing-data-and-etl/[Example: Import Northwind Data Set]
+* link:https://youtu.be/Eh_79goBRUk[Video: LOAD CSV in the Real World^]
 
 [#batch-importer]
-== Super Fast Batch Importer For Huge Datasets
+== Batch Importer For Large Datasets
 
-`LOAD CSV` is great for importing small - medium sized data, i.e. up to the 10M records range.
-For large data sets, i.e. in the 100B records range, we have access to a specialized bulk importer.
+`LOAD CSV` is great for importing small- or medium-sized data (for instance, up to around 10M records).
+However, for data sets larger than this, we have access to a command line bulk importer.
+The `neo4j-admin import` tool allows you to import CSV data to an empty database by specifying node files and relationship files.
 
-We want to use it to import similar order data into Neo4j: _customers, orders and contained products_.
+We want to use it to import order data into Neo4j: _customers, orders, and ordered products_.
 
-The tool is located in `path/to/neo4j/bin/neo4j-import` and is used as follows:
+The tool is located in `<neo4j-home>/bin/neo4j-admin` and is used as follows:
 
 [source, shell]
 ----
-bin/neo4j-import --into retail.db --id-type string \
-                 --nodes:Customer customers.csv --nodes products.csv  \
-                 --nodes orders_header.csv,orders1.csv,orders2.csv \
-                 --relationships:CONTAINS order_details.csv \
-                 --relationships:ORDERED customer_orders_header.csv,orders1.csv,orders2.csv
+bin/neo4j-admin import --nodes:Product "import/products.csv"                                          --nodes:Order "import/orders.csv"                                              --relationships:CONTAINS "import/order_details.csv"
 ----
 
 The first few rows of data used for this import look like this:
 
-.customers.csv
-[options="header"]
-|===
-| customerId:ID(Customer) | name
-| 23 | Delicatessen Inc
-| 42 | Delicous Bakery
-|===
-
 .products.csv
 [options="header"]
 |===
-| productId:ID(Product) | name | price | :LABEL
-| 11 | Chocolate | 10 | Product;Food
+| productId:ID,ProductName
+| 1,Chai
+| 2,Chang
 |===
 
-.orders_header.csv,orders1.csv,orders2.csv
+.orders.csv
 [options="header"]
 |===
-| orderId:ID(Order) | date | total | customerId:IGNORE
```

Comment:
this showed how to use a IGNORE header to only use parts of the file, so you can use the saem file with 2 different headers for a multi-pass, once for the entity and once for the relationship

really important if. you have denormalized input CSV

Reply:
Ah, ok. I'll see if I can get that back in there.